src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -0,0 +1,359 @@
+package org.isf.medicalinventory.gui;
+
+import org.isf.utils.jobjects.InventoryState;
+import org.isf.generaldata.MessageBundle;
+import org.isf.utils.jobjects.ModalJFrame;
+
+import com.toedter.calendar.JDateChooser;
+
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import javax.swing.JScrollPane;
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableModel;
+
+import java.awt.Dimension;
+import javax.swing.JFrame;
+import java.awt.GridBagLayout;
+import java.awt.GridBagConstraints;
+import java.awt.Insets;
+
+import javax.swing.JComboBox;
+import javax.swing.SwingConstants;
+import javax.swing.border.EmptyBorder;
+
+public class InventoryWardBrowser extends ModalJFrame {
+
+    private static final long serialVersionUID = 1L;
+
+    private JDateChooser jCalendarTo;
+    private JDateChooser jCalendarFrom;
+    private GregorianCalendar dateFrom = new GregorianCalendar();
+    private GregorianCalendar dateTo = new GregorianCalendar();
+    private JLabel jLabelTo;
+    private JLabel jLabelFrom;
+    private JPanel panelHeader;
+    private JPanel panelFooter;
+    private JPanel panelContent;
+    private JButton closeButton;
+    private JButton newButton;
+    private JButton updateButton;
+    private JButton deleteButton;
+    private JButton viewButton;
+    private JScrollPane scrollPaneInventory;
+    private JTable jTableInventory;
+    private String[] pColums = { MessageBundle.getMessage("angal.inventory.referenceshow"),
+            MessageBundle.getMessage("angal.inventory.ward"), MessageBundle.getMessage("angal.common.date.txt"),
+            MessageBundle.getMessage("angal.inventory.state"), MessageBundle.getMessage("angal.inventory.user") };
+    private int[] pColumwidth = { 150, 150, 100, 100, 150 };
+    private JComboBox<Object> stateComboBox;
+    private JLabel stateLabel;
+
+    public InventoryWardBrowser() {
+        initComponents();
+    }
+
+    private void initComponents() {
+
+        setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
+        setMinimumSize(new Dimension(850, 550));
+        setLocationRelativeTo(null); // center
+        setTitle(MessageBundle.getMessage("angal.inventory.managementwardtitle"));
+
+        panelHeader = getPanelHeader();
+        getContentPane().add(panelHeader, BorderLayout.NORTH);
+
+        panelContent = getPanelContent();
+        getContentPane().add(panelContent, BorderLayout.CENTER);
+
+        panelFooter = getPanelFooter();
+        getContentPane().add(panelFooter, BorderLayout.SOUTH);
+
+        ajustWidth();
```

Comment:
what is the purpose of making a small private routine that is called once?  Why not just inline the code here.
BTW, the method should be spelled: **adjustwidth**